_test/test_utilities_herbert/test_replicate_array.m

```diff
@@ -0,0 +1,344 @@
+classdef test_replicate_array < TestCase
+    % Test behaviour of replicate_array, replicate_iarray and replicate_logarray.
+    % The original algorithms of thes functions have been superseded by the
+    % Matlab intrinsic function repelem.
+    
+    methods
+        %--------------------------------------------------------------------------
+        % Test replicate_array
+        %--------------------------------------------------------------------------
+        function test_array_empty_v_empty_n (~)
+            % Empty input array, empty n ==> zerp(0,1) output
```

Comment:
```suggestion
            % Empty input array, empty n ==> zero(0,1) output
```
the zerp function is too important for this

Reply:
A quick search reveals that there is no publicly known zerp function. We have the opportunity to create it ourselves and launch it on an expectant world.